sdk/core/azure-core-test/TestProxyMigrationGuide.md

```diff
@@ -148,6 +149,21 @@ made to `https://REDACTED-secondary.table.core.windows.net`, and URLs will also
 
 For more details about sanitizers and their options, please refer to [TestProxySanitizer][test_proxy_sanitizer].
 
+### Removing sanitizers
+Azure SDK tools now include a [common set of sanitizers](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs) applied by default to all test cases.
+These sanitizers are designed to remove sensitive information from the test recordings.
+Users have the option to disable specific sanitizers if needed.
+To disable a sanitizer, the `interceptorManager.removeSanitizer("SanitizerID")` method can be used within the test setup. 
+This flexibility allows developers to customize the sanitization process according to their testing requirements.
+
+- For example, to remove the sanitizer that targets the `$..id` body key, one would use `interceptorManager.removeSanitizer("AZSDK3440")`.
+```java
+if (!interceptorManager.isLiveMode()) {
+    // Remove the default sanitizer that redacts the value of JSON key "id" from the response body
+    interceptorManager.removeSanitizer("AZSDK3440");
```

Comment:
minor suggestion: maybe have the example here remove two just to show the api does that at a glance.